### DISCUSSION TITLE
dark mode in tailwindcss

### DISCUSSION THREAD

**Discussion by soheilhasanjani** [tailwindlabs/tailwindcss#10033] [Help]
Hello, inside the document, the structure you suggested for dark mode is as follows:
```
<div class="bg-white dark:bg-slate-800">...</div>
```
And when the html tag gets the dark class, this will be applied if darkMode: 'class' is configured inside the tailwind config.
There is a way to define a color in tailwind.config.js like this:
```
 colors: {
      brand: {
        deafult: "#000",
        dark: "#fff",
      },
},
```
After adding the dark class to html, it will automatically switch between these two colors?

**Comment by soheilhasanjani** [comment]
@adamwathan

**Comment by thislooksfun** [comment]
With `darkMode` set to `"class"`, any `dark:` styles will be used whenever the dark mode class (defaults to `"dark"`) is anywhere in the parent tree. There is no built-in way to have colors change the way you show. If you want to have something have a white background in light mode and a black one in dark mode, use `class="bg-white dark:bg-black"`.
